Build.cs

```diff
@@ -289,15 +289,9 @@ private void WriteItems()
                 bool isCycleRepeatable = Items.CYCLE_REPEATABLE.Contains(itemId);
                 int replacesItemId = ItemList[i].ReplacesItemId;
 
-                if (ItemUtils.IsItemDefinedPastAreas(itemId)) {
-                    // Subtract amount of entries describing areas and other
-                    itemId -= Values.NumberOfAreasAndOther;
-                }
-
-                if (ItemUtils.IsItemDefinedPastAreas(replacesItemId)) {
-                    // Subtract amount of entries describing areas and other
-                    replacesItemId -= Values.NumberOfAreasAndOther;
-                }
+                itemId -= ItemUtils.GetItemOffset(itemId);
+
+                replacesItemId -= ItemUtils.GetItemOffset(replacesItemId);
```

Comment:
Not really that related to your changes, but is there any good reason why we mix items and areas? Is it for the dependency/conditional checks?

Reply:
Yep exactly for that reason. Might have to rework it one day, but not now.